packages/paste-website/src/pages/experiences/console/index.mdx

```diff
@@ -0,0 +1,379 @@
+---
+title: Console
+description: An overview of the Console experience's design guidelines.
+slug: /experiences/console/
+---
+
+import {graphql} from 'gatsby';
+
+import {Card} from '@twilio-paste/card';
+import {Heading} from '@twilio-paste/heading';
+import {Paragraph} from '@twilio-paste/paragraph';
+import {Stack} from '@twilio-paste/stack';
+import {Table, THead, TBody, Tr, Td, Th} from '@twilio-paste/table';
+import {SidebarCategoryRoutes} from '../../../constants';
+import {ImageCaption} from '../../../components/ImageCaption';
+
+export const pageQuery = graphql`
+  {
+    mdx(frontmatter: {slug: {eq: "/experiences/console/"}}) {
+      fileAbsolutePath
+      frontmatter {
+        slug
+        title
+      }
+      headings {
+        depth
+        value
+      }
+    }
+  }
+`;
+
+<content>
+
+<ComponentHeader
```

Comment:
It looks like the breadcrumb on this page is "Components". Could it instead be "Experiences" unlinked (like we have here for foundations: https://paste.twilio.design/foundations/data-visualization/)?

Same with the Flex, Mobile, Segment, and SendGrid pages.

Reply:
@gloriliale this is super easy to update, you just need to add 'experiences' to this function: https://github.com/twilio-labs/paste/blob/09762f0f1bcfd42d901bd90c33279be68464c68c/packages/paste-website/src/utils/RouteUtils.ts#L52

Reply:
Done! Thanks @shleewhite and @katieporter!